testdata/p4_16_samples_outputs/pna-add-on-miss.p4.spec

```diff
@@ -28,8 +28,10 @@ struct next_hop_arg_t {
 }
 
 struct main_metadata_t {
-	bit<32> pna_main_input_metadata_input_port
-	bit<32> pna_main_output_metadata_output_port
+	;oldname:pna_main_input_metadata_input_port
+	bit<32> pna_main_input_metadata_input0
```

Comment:
This shouldn't change and likewise many other spec files

Reply:
currently we allowing only 30 char, so it's changing.